config/config_env.go

```diff
@@ -183,7 +183,7 @@ func reporterConfigFromEnv() (*ReporterConfig, error) {
 // is an environment variable and `defaultValue` is the value to use in case the env var is not set
 func parseTags(sTags string) []opentracing.Tag {
 	pairs := strings.Split(sTags, ",")
-	tags := make([]opentracing.Tag, 0)
+	var tags []opentracing.Tag
```

Comment:
lint was complaining about this; why doesnt the build fail if lint fails?

Reply:
which lint?